### PR TITLE
feat: added list notes and delete case commands and fix: removed unused import "timedelta"

### PR DESCRIPTION
**Pull request by ChiliMX**

### CODE REVIEW

File: cogs/moderation/moderation.py [hunk 1]
```diff
@@ -219,18 +227,69 @@ async def _unban(self, ctx, user: discord.Object, *, reason="No Reason Provided"
...
-                                  ctx.author.id, str(member), str(ctx.author), "note", content)
+
+        await self.bot.db.execute('''INSERT INTO Cases(id, guildid, userid, modid, username, modname, case_type, case_data, created_at)
+                                  VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)''', cid, ctx.guild.id, member.id,
+                                  ctx.author.id, str(member), str(ctx.author), "note", content, datetime.now())
 
         """
         TODO:
         create notes command to get a list of notes in a user
         """
```
Comment (ToxicKidz) [inline cogs/moderation/moderation.py]:
```suggestion
```
You already made the notes command.

**Review by ToxicKidz — CHANGES_REQUESTED** [review]

File: cogs/moderation/moderation.py (hunk 1 above)
Comment (ChiliMX) [reply cogs/moderation/moderation.py]: fixed that now

**Review by ToxicKidz — CHANGES_REQUESTED** [review]
@ChiliPy Resolve conflicts and then it can be merged.

**Comment by ToxicKidz** [comment]
This is taking too long so imma close this.

**Pull request closed without merging**